packages/quantic/force-app/main/default/lwc/quanticResultTemplate/__tests__/quanticResultTemplate.test.js

```diff
@@ -0,0 +1,90 @@
+// @ts-ignore
+import QuanticResultTemplate from '../quanticResultTemplate';
+import {buildCreateTestComponent, cleanup, flushPromises} from 'c/testUtils';
+
+const createTestComponent = buildCreateTestComponent(
+  QuanticResultTemplate,
+  'c-quantic-result-template'
+);
+
+const selectors = {
+  slotByName: (name) => `slot[name="${name}"]`,
+  container: (name) => `div[data-testid="${name}_container"]`,
+};
+
+const previewSlots = ['label', 'badges', 'date', 'visual'];
+const otherSlots = [
+  'actions',
+  'title',
+  'metadata',
+  'emphasized',
+  'excerpt',
+  'bottom-metadata',
+  'children',
+];
+const allSlots = previewSlots.concat(otherSlots);
+
+describe('c-quantic-result-template', () => {
+  afterEach(() => {
+    cleanup();
+  });
+
+  it('should display all slots by default', async () => {
+    const element = createTestComponent();
+    await flushPromises();
+
+    expect(element.shadowRoot.firstChild.className).toBe(
+      'lgc-bg slds-p-vertical_medium slds-border_bottom'
```

Comment:
maybe you can store that and give it a meaningful name otherwise its kinda hard to tell what this is supposed to represent